### PR TITLE
Add a space between HTML attributes in header

### PR DESCRIPTION
**Pull request by nalysius**
The links added to the menu were missing spaces between their attributes. It's now present, making the HTML compliant.

This time it's done on one line, my replacing {{- by {{. It seems the dash is a space controller.

### CODE REVIEW

**Review by 526avijitgupta — APPROVED** [review]
Lgtm but please wait for @yashmehrotra 's approval

**Comment by yashmehrotra** [comment]
Thanks for the PR @nalysius

**Pull request merged**
